index.js

```diff
@@ -12,6 +11,39 @@ const levelMapping = [
 	'ansi16m'
 ];
 
+const stringReplaceAll = (str, substr, replacer) => {
+	let idx = str.indexOf(substr);
+	if (idx === -1) {
+		return str;
+	}
+
+	const subLen = substr.length;
+	let end = 0;
+	let res = '';
+	do {
+		res += str.substr(end, idx - end) + replacer;
+		end = idx + subLen;
+		idx = str.indexOf(substr, end);
+	} while (idx !== -1);
+
+	res += str.substr(end);
```

Comment:
Use `slice` instead of `substr`. See; https://github.com/sindresorhus/eslint-plugin-unicorn/issues/65

Reply:
FWIW Substr is slightly faster in v8.

Reply:
^ that. It is in hot path.

Reply:
Not even mentioning that `substr` is a universally understood operation with functions named that way before even C was invented, which exists in almost all languages, both modern and dated, while `slice` is only used in a handful of languages with semantics differing from place to place.